content/systems/filecoin_blockchain/message_pool/message_syncer.md

```diff
@@ -1,34 +1,32 @@
 ---
-title: Message Syncer
+title: Message Propagation
 weight: 1
 dashboardWeight: 2
 dashboardState: incomplete
 dashboardAudit: 0
 dashboardTests: 0
 ---
 
-# Message Syncer
+# Message Propagation
 ---
 
-Transaction messages are sorted in the `mpool` of miners as they arrive according to cryptoeconomic rules followed by the miner.
+The message pool has to interface with the libp2p pubsub [GossipSub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) protocol. This is because transaction messages are propagated over [GossipSub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) _topics_. Every serialised `SignedMessage` is announced (see [Message](message)) in the corresponding topic by any node participating in the network.
```

Comment:
`Every serialised `SignedMessage` is announced (see [Message](message)) in the corresponding topic by any node participating in the network.`

There might be too much going on on this sentence. Maybe we can drop the serialization details and the struct name and focus more on the topic (maybe even naming it fully).

Reply:
(Also, minor detail, mentioning "topics" in plural might be confusing. I get that we're referring to a generic description, but it's a single topic, and using a generic unnamed plural forces later the use of the "corresponding" qualifier that doesn't add much more information because, for example, I don't know if there is a corresponding topic for different types of messages, which is not the case.)